### ISSUE TITLE
"Commands" tab scroll bar resets after click

### ISSUE THREAD

**Issue by mac-pfa** [jupyterlab/jupyterlab#3923]
When an option in the "Commands" tab is clicked, the scroll bar resets to the top of the list of options

**Comment by saulshanabrook** [comment]
Yep, that's not very helpful. Makes it hard to execute a bunch of commands from the same section one after another. I assume this is a bug and not intentional.

**Comment by sccolbert** [comment]
Does this happen when the search field is empty before you click?

**Comment by saulshanabrook** [comment]
@sccolbert Yeah, for it resets regardless of whether these is something in the search field or not.
